src/main/java/eu/europeana/metis/sandbox/config/amqp/CreatedQueueConfig.java

```diff
@@ -12,23 +12,23 @@
  * The type Created queue configuration.
  */
 @Configuration
-public class CreatedQueueConfiguration extends QueueConsumerConfiguration {
+public class CreatedQueueConfig extends QueueConsumerConfig {
 
-  @Value("${sandbox.rabbitmq.queues.record.created.consumers:12}")
+  @Value("${sandbox.rabbitmq.queues.record.created.consumers}")
```

Comment:
the default value is a good to have in case of misconfiguration.  `sandbox.rabbitmq.queues.record.created.consumers:12`, can we set it then to two? like `sandbox.rabbitmq.queues.record.created.consumers:2` also I see where the confusion began. 
the name of the property was changed. original in the sample yml was
```
consumers: 12
max-consumers: 12
prefetch: 3
```
and I see it mutated to
```
concurrency: 12
max-concurrency: 12
prefetch: 3
```

Reply:
by mistake also was copy pasted the same way to the docker in test environment because in production is not like this.